README.md

```diff
@@ -8,12 +8,10 @@
 [![Gitter](https://badges.gitter.im/amfoss/TempleApp.svg)](https://gitter.im/amfoss/TempleApp)
 [![Code of Conduct](https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-e04545.svg?style=flat)](https://github.com/amfoss/TempleApp/blob/master/CODE_OF_CONDUCT.md)
 [![Travis CI](https://travis-ci.com/amfoss/TempleApp.svg?branch=master)](https://travis-ci.com/amfoss/TempleApp)
-[![Open Issues](https://img.shields.io/github/issues/amfoss/TempleApp.svg)](https://GitHub.com/amfoss/TempleApp/issues/)
-[![PRs](https://img.shields.io/github/issues-pr/amfoss/TempleApp.svg)](https://GitHub.com/amfoss/TempleApp/pulls/)
-[![Contributors](https://img.shields.io/github/contributors/amfoss/TempleApp.svg)](https://GitHub.com/amfoss/TempleApp/graphs/contributors/)
 
-![Android Studio](https://img.shields.io/badge/android%20studio-v3.4.1-blue.svg?cacheSeconds=2592000)
-![Gradle](https://img.shields.io/badge/gradle-v5.1.1-green.svg?cacheSeconds=2592000)
+
+[![Android Studio](https://img.shields.io/badge/android%20studio-v3.4.1-blue.svg?cacheSeconds=2592000)](https://developer.android.com/studio/?gclid=Cj0KCQjwgezoBRDNARIsAGzEfe58cXcPjI00Y3vzUdGlJ6FzVv6gzdmO_xCuI8kEBGCXwlnePyBOk6gaAvMXEALw_wcB)
```

Comment:
why is the query gclid needed? i guess it won't be requied

Reply:
Yeah not needed.

Reply:
updated.